src/main/java/pages/ProductsPage.java

```diff
@@ -64,6 +67,7 @@ public ProductsPage clickAddOrRemoveProductItem(String productName, boolean isAd
      * @return product price
      */
     public String getProductPrice(String productName) {
+        AllureUtils.takeScreenshot(driver);
```

Comment:
I advise you to take screenshots only when test fails (using TestListener), there is no need in screenshots when tests are passed for example

Reply:
done